tests/test_models.py

```diff
@@ -100,25 +100,25 @@ def test_robots_page_no_site(self):
         extension = PageSitemapProperties.objects.create(
             extended_object=page1, priority='0.2', changefreq='never'
         )
-        page1.publish('en')
+        #page1.publish('en')
```

Comment:
Don't comment these out, please delete them, there's no publishing in CMS 4 without djangocms versioning.